custom/icds_reports/utils/__init__.py

```diff
@@ -1590,3 +1593,25 @@ def get_datatables_ordering_info(request):
 
 def phone_number_function(x):
     return "+{0}{1}".format('' if str(x).startswith('91') else '91', x) if x else x
+
+
+def filter_cas_data_export(export_file, location):
+    loc_type = location.location_type.name
```

Comment:
nit: `loc_type_name` might be a little clearer

Reply:
also looks like this is only used in one place so you could just use `location.location_type.name` in the f-string (assuming it's not a performance hit)